tests/Types/TimeImmutableTypeTest.php

```diff
@@ -105,7 +105,7 @@ public function testResetDateFractionsWhenConvertingToPHPValue(): void
 
         $date = $this->type->convertToPHPValue('15:58:59', $this->platform);
 
-        self::assertSame('1970-01-01 15:58:59', $date->format('Y-m-d H:i:s'));
+        self::assertSame('1970-01-01 15:58:59', $date?->format('Y-m-d H:i:s'));
```

Comment:
This and similar changes should be reverted. We don't expect `$date` to be NULL here.

Reply:
I admit, this was to make static analysis happy. Now that the return type is not `mixed` anymore, the tools started to analyze this piece of code. Shall I add an additional `assertNotNull()` instead?

Reply:
> Shall I add an additional `assertNotNull()` instead?

Yeah, that would be the right solution.